dev/CONTRIBUTING.md

```diff
@@ -100,87 +94,60 @@ Run `builder run check` before committing to ensure lint and tests are passing.
-- Have sensible default props / fallback behaviors so that _something_ is rendered even when no props are provided
-- Produce sensible results if _only_ data is provided _i.e_ set the domain based on whatever data was provided
-- Have attractive default styling
-- Take a `style` prop, and intelligently merge provided (and scoped) styles with default styles
-
-```js
-// default styles
-const styles = {
-  parent: {
-    width: 500,
-    height: 300,
-    margin: 50
-  },
-  data: {
-    fill: "#756f6a",
-    opacity: 1,
-    stroke: "transparent",
-    strokeWidth: 0
-  },
-  labels: {
-    stroke: "none",
-    fill: "black",
-    fontFamily: "Helvetica",
-    fontSize: 10,
-    textAnchor: "middle"
-  }
-};
+## Victory Component Style Guide
 
-// merging styles:
-getStyles(props) {
-  if (!props.style) {
-    return styles;
-  }
-  const {data, labels, parent} = props.style;
-  return {
-    parent: _.merge({}, styles.parent, parent),
-    labels: _.merge({}, styles.labels, labels),
-    data: _.merge({}, styles.data, data)
-  };
-}
-
-// example style prop
+Victory is an ecosystem of modular components with similar language and methodologies. Because these components are designed to work together, a consistent set of patterns and conventions is necessary. Please read the following set of standards and examples if you are interested in writing a larger feature, or contributing a new victory component.
+
+### Flexibility
+
+Victory is designed to be as flexible as possible. Code should be free of hard-coded values or one-off solutions. The following patterns also help keep the library flexible: 
+
+**Seperating Rendered Components**
+
+Any rendered component (_i.e._ `<line/>`, `<text/>` and even `<g/>`) should be written as seperate components, and included via `defaultProps` so that they can be overridden the users. This pattern also allows us to use the same code base to support a react native version of Victory. 
+
+These primitive rendered elements are kept in the `victory-core` repo. Please check [here](https://github.com/FormidableLabs/victory-core/tree/master/src/victory-primitives) first before writing a new rendered component. Rendered components should be stateless and as general as possible. [`Point`](https://github.com/FormidableLabs/victory-core/blob/master/src/victory-primitives/point.js) is a good example of a rendered component. Notice that `Point` is written so that it can easily be extended to a [react native compatible version](https://github.com/FormidableLabs/victory-core-native/blob/master/lib/components/victory-primitives/point.js). The appropriate versions of `Point` can then be included as `defaultProps` in [`VictoryScatter`](https://github.com/FormidableLabs/victory-chart/blob/master/src/components/victory-scatter/victory-scatter.js#L353), and the [native version](https://github.com/FormidableLabs/victory-chart-native/blob/master/lib/components/victory-scatter.js)
```

Comment:
- missing a period at the end of this line.
- I'd make this language really clear, so that the reader didn't have to follow the link to see the meaning:
`...defaultProps in *VictoryScatter*, and the *native version*` -> `...defaultProps in the *standard version* and the *native version* of VictoryScatter`.